internal/testing/prometheus_stuffs.yaml

```diff
@@ -68,11 +67,5 @@ metadata:
     name: prometheus-server-conf
   namespace: kube-system
 data:
-  prometheus.yml: |-
-    global:
-      scrape_interval: 5s
-      evaluation_interval: 5s
-    scrape_configs:      
-      - job_name: 'k8s-shredder'
-        static_configs:
-          - targets: ['k8s-shredder.kube-system.svc.cluster.local:8080']
+  prometheus.yml: "global:\n  scrape_interval: 5s\n  evaluation_interval: 5s\nscrape_configs:\
```

Comment:
Nit, this formatting is worse compared with the previous one, maybe the yaml fix can be tweaked to avoid formatting strings like this

Reply:
Got it.  Found and fixed!